pkg/render/logstorage/linseed/linseed.go

```diff
@@ -97,6 +99,8 @@ type Config struct {
 	// Whether this is a management cluster
 	ManagementCluster bool
 
+	MultiTenant bool
```

Comment:
We don't need an explicit option - the code can just check `if c.cfg.Tenant != nil`